frontend/src/app.module.css

```diff
@@ -28,15 +28,58 @@
   border-radius: 24px;
   background-color: var(--white);
   overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
 }
 
 .sidepanel {
   position: relative;
-  height: 100%;
+  max-height: 100%;
+  overflow-x: hidden;
+  word-wrap: break-word;
+  padding-left: 8px;
+  padding-right: 24px;
+  font-size: 16px;
+  overflow-y: auto;
+  flex: 1;
 }
 
 .close_container {
-  position: absolute;
-  top: -32px;
-  right: -16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 56px;
+  margin: -16px;
```

Comment:
Any simple alternatives to negative margins here?

Reply:
I'm working on this now. The reason I added it is to make sure the padding of the column element doesn't affect the report title. This way, the UI lines up properly with the UX specs.